pkg/forwarder/interceptor/interceptor.go

```diff
@@ -46,7 +53,73 @@ func NewInterceptor(agentSocket, nsPath string) Interceptor {
 		return conn, nil
 	}
 
-	interceptor := agentproto.NewRedirector(agentDialer)
+	redirector := agentproto.NewRedirector(agentDialer)
+
+	return &interceptor{
+		Redirector: redirector,
+	}
+}
+
+func (i *interceptor) CreateContainer(ctx context.Context, req *pb.CreateContainerRequest) (*types.Empty, error) {
+
+	logger.Printf("CreateContainer: containerID:%s", req.ContainerId)
+
+	res, err := i.Redirector.CreateContainer(ctx, req)
+
+	if err != nil {
+		logger.Printf("CreateContainer fails: %v", err)
```

Comment:
nit: wondering if replacing "fails" with "failed with error" be better ?

Reply:
OK. I'll change the message.